### DISCUSSION TITLE
When hosting, I have already brought down this project for hosting in a subdirectory https://domain.com/ms/guide , can't get to js files and styles, writes 404

### DISCUSSION THREAD

**Discussion by ghost** [vuejs/vitepress#609] [Q&A]
When hosting, I have already brought down this project for hosting in a subdirectory https://domain.com/ms/guide , can't get to js files and styles, writes 404. How can I specify the path to styles and js files when building
![error1](https://user-images.githubusercontent.com/58338625/165007891-da912638-fc87-43a4-a466-a9aa928ddf61.jpg)

Can I somehow register the path to the files myself within vitepress?

**Comment by poziminski** [comment]
There is a `base` setting you should use: https://vitepress.vuejs.org/config/app-configs#base
